javascript/javascript1/week3/noteApp.js

```diff
@@ -0,0 +1,50 @@
+const notes = [];
+
+function saveNote(content, id) {
+  // write some code here
+  let noteInfo = {
+      content:content,
+      id:id
```

Comment:
There's a shorthand for adding properties to objects, when they have the same variable and key name like you have.

```javascript
const noteInfo = {
  content, // same as "content: content,"
  id // same as "id: id"
}
```

Reply:
ok, this is good thing , i see . Ok thanks for the logic.